### DISCUSSION TITLE
User Scheduling for Activation

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#623] [Suggestions]
Another feature I would like to see added to the shop system is for players to schedule the activation of their own packs, either by setting their own times and dates where the pack will activate (and, if configured, expire based on its own set-up), or using pre-made time slots that users can pick and choose to have activate at those slots. (I also see a potential of integrating this with the Community Website Calendar with event creation.)

**Comment by mvaessen** [comment]
Not sure i understand what this suggestion is about. 
Can you perhaps post an example?

**Comment by mvaessen** [comment]
going to close due to missing examples
